Library/Homebrew/formula.rb

```diff
@@ -1981,6 +1981,11 @@ def setup_home(home)
       import site; site.addsitedir("#{HOMEBREW_PREFIX}/lib/python2.7/site-packages")
       import sys, os; sys.path = (os.environ["PYTHONPATH"].split(os.pathsep) if "PYTHONPATH" in os.environ else []) + ["#{HOMEBREW_PREFIX}/lib/python2.7/site-packages"] + sys.path
     PYTHON
+
+    # Don't let bazel write to tmp directories we don't control or clean.
+    (home/".bazelrc").write <<~EOS
+      startup --output_user_root=#{home}/_bazel
+    EOS
```

Comment:
```suggestion
    (home/".bazelrc").write "startup --output_user_root=#{home}/_bazel"
```
Not a strong preference, but you probably don't need a heredoc here, do you? Though I guess the difference is that a heredoc writes a new line at the end of the file.

Reply:
Yeah, I think I was going to do more but then realised I didn't. Somewhat surprised Ruby doesn't add a newline for the one-line version, since a newline at the end of the file is POSIX standard.

Reply:
Ah, yes. It's `puts` not `write` which does that - but `Pathname` only has a `write` method.

Reply:
```suggestion
    (home/".bazelrc").write "startup --output_user_root=#{home}/_bazel\n"
```

feels nice if you really want the newline. Does Bazel fail without it?

Reply:
Haven't tested yet. It's maybe ok with it.